### ISSUE TITLE
Ablate the learning rate for attentive probe

### ISSUE THREAD

**Issue by robertdvdk** [jwpartyka/uva-fomo#22]
Currently, the attentive probe has to be trained for quite a while, even though it has few parameters. We can try to increase the learning rate so we can use fewer epochs.
